### ISSUE TITLE
ofxDarknet vs darknet app

### ISSUE THREAD

**Issue by marc-antimodular** [mrzl/ofxDarknet#10]
My compiled version of the yolo ofxDarknet demo does not behave like the darknet demo application.
At startup, the yolo ofxDarknet demo shows tons of labels, then after a few seconds, no labels at all.
This dysfunctional behavior occurs with all yolo configs. What could be wrong?

**Comment by genekogan** [comment]
the first line in `draw` is controlling the threshold of what labels to include with the mouse: `float thresh = ofMap( ofGetMouseX(), 0, ofGetWidth(), 0, 1 );` maybe you are adjusting it all the way to the right?

**Comment by marc-antimodular** [comment]
You are right; it works as you described.

But it crashes after a few minutes of use, sometimes crashing the computer. I got messages about low memory (my computer have 16G of RAM). How different is this version from the original software?

**Comment by genekogan** [comment]
i haven't had this issue but i haven't used it several minutes at a time. @mrzl mentioned experiencing something like this too, possibly because of a memory leak which we need to investigate.

**Comment by schwittlick** [comment]
Indeed, I was experiencing the same. Even after investigating for ages, I couldn't track down the obvious memory leak. I will update here once I found and fixed it..

**Comment by gurmeetsidhu** [comment]
uhmmm ... not sure if this is the fix but when I was working with training darknet using another application I had to increase my sub-divisions in the cfg file double and then that kind of stopped the memory errors. If you're video ram is leading to the issue that may fix it.

**Comment by gurmeetsidhu** [comment]
@mrzl Have you resolved this memory leak I haven't been able to find a memory leak or lower its effects either

**Comment by genekogan** [comment]
@gurmeetsidhu you're making reference to a different issue, which is running out of memory on startup. increasing subdivisions as you say fixed that for us (and we updated the cfg files for that). i often have to close a lot of applications as well (especially those which consume vram) to get it to work sometimes.

@marc-antimodular the problem you described in january should also be fixed. it was due to a memory leak which we fixed in [https://github.com/mrzl/ofxDarknet/commit/db47306fa7b34d121533afe9c4f13b6bb814f5c4](https://github.com/mrzl/ofxDarknet/commit/db47306fa7b34d121533afe9c4f13b6bb814f5c4). sorry for the late notice, hope it's useful.
